test/PowerShellEditorServices.Test.E2E/LanguageServerProtocolMessageTests.cs

```diff
@@ -1057,18 +1120,20 @@ public async Task CanSendGetCommandRequest()
         [SkippableFact]
         public async Task CanSendExpandAliasRequest()
         {
+            throw new Exception("asdf");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5596138)